packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -14,7 +14,8 @@ console.debug({ CURRENT_ENGINES_HASH })
 const FIXED_ENGINES_HASH = 'da41d2bb3406da22087b849f0e911199ba4fbf11'
 const dirname = process.platform === 'win32' ? __dirname.split(path.sep).join('/') : __dirname
 
-jest.setTimeout(200_000)
+// Network can be slow, especially for macOS in CI.
+jest.setTimeout(300_000)
```

Comment:
Should we increase the timeout programmatically in some target systems, like `darwin`, then?

Reply:
It can be slow regardless of the system so a global increase is ok. Also we are not benchmarking our CDN, just having a "reasonable limit"

Reply:
This is the download everything case, which is a lot and totally not what a user would experience.